docs/develop-resources/deep-dives/4-add-controller.md

```diff
@@ -5,6 +5,8 @@ Run the following command to generate a controller template
 ```
 cd dev/tools/controllerbuilder
 go run main.go generate-controller --service <YOUR_SERVICE> --api-version <VERSION> --kind <YOUR_RESOURCE> --proto-resource <PROTO_RESOURCE>
+# For example,
+# go run main.go generate-controller --service alloydb.v1beta --api-version alloydb.cnrm.cloud.google.com/v1beta1 --kind AlloyDBInstance --proto-resource instance
```

Comment:
The flag values should align with those used in other tools (as detailed in Step 2). If there is any discrepancy, we should get it fixed.

cc @yuwenma maybe we should combine `--kind` and `--proto-resource` into one flag, similar to what we did for the type generator.

Reply:
nit: move the last line in its own bash code snippet.

Reply:
> The flag values should align with those used in other tools (as detailed in Step 2). If there is any discrepancy, we should get it fixed.
> 
> cc @yuwenma maybe we should combine `--kind` and `--proto-resource` into one flag, similar to what we did for the type generator.

+1. I thought the flag aligned already (?)